src/components/Datatable/Table/TableCell/TableCell.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const StyledTableCell = styled.td`
-  padding: 5px;
-  background-color: #fff;
+import { getColor, pxToRem } from '../../../../utils/helpers';
+import { TableCellProps } from './TableCell.types';
+
+const StyledTableCell = styled.td<{ $isOdd: boolean }>`
+  padding: ${pxToRem(12, 8)};
   display: flex;
   align-items: center;
-  border-bottom: 1px solid #dadada;
+  border-bottom: 1px solid ${getColor('graphiteH')};
   border-right: 1px solid transparent;
+  background-color: ${({ $isOdd, theme }) =>
+    $isOdd ? '#fcfcfc' : getColor('graphite5H', { theme })};
```

Comment:
Pls, do you know if new Zebra color is just one time use for data table? Most likely yes, eventually we can elevate it later on

Reply:
It's one time only, checked with Mate